rtos/ThisThread.h

```diff
@@ -174,11 +174,17 @@ void sleep_until(uint64_t millisec);
 void yield();
 
 /** Get the thread id of the current running thread.
-  @return  thread ID for reference by other functions or NULL in case of error or in ISR context.
+  @return  thread ID for reference by other functions or NULL in case of error.
   @note You may call this function from ISR context.
 */
 osThreadId_t get_id();
 
+/** Get the thread name of the current running thread.
+  @return  thread name pointer or NULL in case of error or in ISR context.
```

Comment:
Could also be NULL if the thread has no name. (Possible in CMSIS-RTOS, even though the `Thread` object likes to put in `"application_unnamed_thread"`)